kernel/src/table_changes/scan.rs

```diff
@@ -198,6 +213,7 @@ mod tests {
     use crate::expressions::{column_expr, Scalar};
     use crate::scan::ColumnType;
     use crate::schema::{DataType, StructField, StructType};
+    use crate::table_changes::COMMIT_VERSION_COL_NAME;
```

Comment:
can just do use super::*?

Reply:
Doesn't seem like it. `super` would point to the scan module, but we need its parent.